servers/rendering/rendering_device.cpp

```diff
@@ -3181,6 +3188,7 @@ RID RenderingDevice::uniform_set_create(const Collection &p_uniforms, RID p_shad
 		RDD::BoundUniform &driver_uniform = driver_uniforms[i];
 		driver_uniform.type = uniform.uniform_type;
 		driver_uniform.binding = uniform.binding;
+		driver_uniform.ids.reserve(uniform.get_id_count());
```

Comment:
Are you sure the `ids` are empty at this point?

Reply:
The whole `driver_uniforms` is defined just above this loop and resized, so all these items are just default constructed, unless you do some weird stuff in constructor it should be empty.